heat/core/memory.py

```diff
@@ -22,3 +24,44 @@ def copy(a):
     return dndarray.DNDarray(
         a._DNDarray__array.clone(), a.shape, a.dtype, a.split, a.device, a.comm
```

Comment:
can you add a.order to this?

Reply:
hm, but order is no tensor attribute. I'm not sure I see the need for this. The way to check the memory layout would be to call a.stride() (or a.strides if you want to be numpy-like)